components/helm-broker/cmd/broker/main.go

```diff
@@ -49,7 +52,14 @@ func main() {
 		ybundle.NewHTTPRepository(cfg.Repository),
 		ybLoader, sFact.Bundle(), sFact.Chart(), log,
 	)
-	err = cachePop.Init()
+	for i := 0; i < maxPopulatorRetries; i++ {
```

Comment:
could you put retry loop inside cachePop.Init() function and pass maxPopulatorRetries to it?